packages/authorization/src/__tests__/acceptance/authorization.options.acceptance.ts

```diff
@@ -0,0 +1,170 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/authorization
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {AuthenticationBindings} from '@loopback/authentication';
+import {Context, invokeMethod, Provider} from '@loopback/context';
+import {Application} from '@loopback/core';
+import {expect} from '@loopback/testlab';
+import {
+  AuthorizationComponent,
+  AuthorizationDecision,
+  authorize,
+  Authorizer,
+} from '../..';
+import {AuthorizationTags} from '../../keys';
+import {AuthorizationOptions} from '../../types';
+
+const Deny = AuthorizationDecision.DENY;
+const Allow = AuthorizationDecision.ALLOW;
+const Abstain = AuthorizationDecision.ABSTAIN;
+
+type DecisionMatrix = [
+  AuthorizationOptions,
+  AuthorizationDecision[],
+  AuthorizationDecision,
+][];
+
+// Decisions not impacted by options.precedence or options.defaultDecision
+const matrix1: DecisionMatrix = [
+  [{}, [Deny, Deny, Deny], Deny],
+  [{}, [Allow, Allow, Allow], Allow],
+  [{}, [Abstain, Allow, Abstain], Allow],
+  [{}, [Abstain, Deny, Abstain], Deny],
+
+  [{defaultDecision: Deny}, [Deny, Deny, Deny], Deny],
+  [{defaultDecision: Deny}, [Allow, Allow, Allow], Allow],
+  [{defaultDecision: Deny}, [Abstain, Allow, Abstain], Allow],
+  [{defaultDecision: Deny}, [Abstain, Deny, Abstain], Deny],
+
+  [{defaultDecision: Allow}, [Deny, Deny, Deny], Deny],
+  [{defaultDecision: Allow}, [Allow, Allow, Allow], Allow],
+  [{defaultDecision: Allow}, [Abstain, Allow, Abstain], Allow],
```

Comment:
one more for
`[{defaultDecision: Allow}, [Abstain, Abstain, Abstain],ALLOW]`
?